### ISSUE TITLE
SUGGESTION: Ability to apply Paxel charms onto already charmed Paxels

### ISSUE THREAD

**Issue by SamayaNL** [Iskallia/Vault-public#803]
Because oftentimes you get charms that arent all that useful to you, you end up with a bunch of paxels you'll never use. I propose the ability to apply charms to paxels whether or not they've been charmed already.

**Comment by bmemike** [comment]
Paxel charms giving random effects is working exactly as intended. 

Paxels are already a huge improvement over normal pickaxes and can take all enchantments besides mending. They have very high durability to offset this. 

If you choose to enter the paxel charm lottery, you're expected to live with the results.

**Comment by SamayaNL** [comment]
> Paxel charms giving random effects is working exactly as intended.
> 
> Paxels are already a huge improvement over normal pickaxes and can take all enchantments besides mending. They have very high durability to offset this.
> 
> If you choose to enter the paxel charm lottery, you're expected to live with the results.

I'm not really sure what your issue is. Right now I've got a bunch of unused paxels which I'm never gonna use. What would be the problem with being able to "reroll" them with more charms? The only difference is you don't have to craft another paxel, you're still using your paxel charms.

**Comment by bmemike** [comment]
I'm confused by your first sentence. I don't have "an issue". I'm responding from the perspective of game design.

Crafting a paxel is a cost. And it's one you need to pay each time you want to try to get some of the rare and powerful paxel charm enchantments.

If there wasn't a clear intention that you would get some paxels that weren't good (they're all "usable"), then you wouldn't have any lesser desirable effects in the charm pool to begin with.

There are many examples of similar mechanics. Take Banished Souls - you apply them to an idol for chance to get higher durability. But in doing so you risk a 33% chance that the idol is destroyed. Just *GONE*.

At least you still have a paxel that you can use here.

**Comment by SamayaNL** [comment]
I mean that you have an issue with my suggestion, thats all. I do have to disagree with your statement that all Paxels are usable. I've got like 5 smelting paxels, and a couple destructive ones. These have active abilities which I just don't want to use. 

Yeah no I understand Iskall wants there to be a trade-off, a risk when applying something. However you do even when you can reroll Paxels. You still have a change of rolling something you dont want, plus you lose a Paxelcharm. I think that's a good enough trade-off for the ability to get something good.

**Comment by bmemike** [comment]
Auto-smelt is a great way to farm charcoal and doesn't require any fuel. Destructive is incredible if you're in a digsite, for example.

There is definitely a lot of application for these paxels if you're just a little bit creative. 

You can obviously choose to or not to use them as you wish, but at least consider that while they may not be top tier, they still have utility.

**Comment by SamayaNL** [comment]
Yes I can see their uses in general (except the destructive paxel, which I assumed, I think rightly so, would destroy any block, including chests, if it doesn't that would make it more useful). However for me, they don't, or don't anymore. I also don't need that many of 'em even it they were useful. I checked the weight of every charm in the table, and smelting has an insanely high chance of rolling. 

We're not gonna agree on this, but I just don't see the downside to changing the way this works. You're still gonna lose a Charm, but you don't have to spend the larimar on paxels you're not gonna use, when you could spend it on catalysts instead. It would be a QOL change which would make the game less frustrating for me. But, agree to disagree.
